examples/light-switch-app/silabs/include/AppEvent.h

```diff
@@ -49,6 +56,10 @@ struct AppEvent
             uint8_t Action;
             int32_t Actor;
         } LightEvent;
```

Comment:
we can delete the light event for the light-switch if it is not used

Reply:
deleted